internal/client/consensus/grandpa/finality_proof_test.go

```diff
@@ -0,0 +1,400 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
```

Comment:
Same for these files, not sure why they are showing up as new files vs moved files

Reply:
I think cause I renamed this file.